src/core/gestures/MdGestureConfig.ts

```diff
@@ -5,22 +5,46 @@ import {HammerGestureConfig} from 'angular2/src/platform/browser_common';
 @Injectable()
 export class MdGestureConfig extends HammerGestureConfig {
   /* List of new event names to add to the gesture support list */
-  events: string[] = ['drag', 'longpress'];
+  events: string[] = [
+    'drag',
+    'dragstart',
+    'dragend',
+    'dragright',
+    'dragleft',
+    'longpress',
+  ];
 
   /*
-  * Overrides default recognizer event names and thresholds.
-  *
-  * Our gesture names come from the Material Design gestures spec:
-  * https://www.google.com/design/spec/patterns/gestures.html#gestures-touch-mechanics
-  *
-  * More information on default recognizers can be found in Hammer docs:
-  * http://hammerjs.github.io/recognizer-pan/
-  * http://hammerjs.github.io/recognizer-press/
-  *
-  * TODO: Confirm threshold numbers with Material Design UX Team
-  * */
-  overrides: {[key: string]: Object} = {
-    'pan': {event: 'drag', threshold: 6},
-    'press': {event: 'longpress', time: 500}
-  };
+   * Builds Hammer instance manually to add custom recognizers that match the Material Design spec.
+   *
+   * Our gesture names come from the Material Design gestures spec:
+   * https://www.google.com/design/spec/patterns/gestures.html#gestures-touch-mechanics
+   *
+   * More information on default recognizers can be found in Hammer docs:
+   * http://hammerjs.github.io/recognizer-pan/
+   * http://hammerjs.github.io/recognizer-press/
+   *
+   * TODO: Confirm threshold numbers with Material Design UX Team
+   * */
+  buildHammer(element: HTMLElement) {
+    var mc = new Hammer(element);
```

Comment:
is there a reason you're using `var` instead of `let`? Just wondering because I've seen `let` being used in other places